ci-operator/config/cri-o/cri-o/cri-o-cri-o-main__ci.yaml

```diff
@@ -6,26 +11,41 @@ build_root:
     tag: golang-1.20
 images:
 - dockerfile_literal: |
-    FROM registry.fedoraproject.org/fedora:38
+    FROM base
+    RUN dnf install -y jq git openssl python3.11 && \
+        ln -sf /usr/bin/python3.11 /usr/bin/python3 && \
+        dnf clean all
     RUN curl https://dl.google.com/dl/cloudsdk/release/google-cloud-sdk.tar.gz > /tmp/google-cloud-sdk.tar.gz
+    RUN python3 --version
     RUN mkdir -p /usr/local/gcloud \
       && tar -C /usr/local/gcloud -xvf /tmp/google-cloud-sdk.tar.gz \
-      && /usr/local/gcloud/google-cloud-sdk/install.sh \
       && rm -f /tmp/google-cloud-sdk.tar.gz
     ENV PATH $PATH:/usr/local/gcloud/google-cloud-sdk/bin
     WORKDIR /src
     COPY . .
-  from: src
+  from: base
   to: crio-crio-base-src
+- dockerfile_literal: |
+    FROM base
+    RUN dnf install -y jq git openssl python3.11 google-cloud-cli-447.0.0-1 gzip jq unzip openssh-clients openssl bind-utils util-linux xz && \
+        ln -sf /usr/bin/python3.11 /usr/bin/python3 && \
```

Comment:
For what do we need this symlink?

Reply:
Python 3.9 is currently the default, so we need to ensure that the `google-cloud SDK` picks the latest version of Python.

Reply:
https://github.com/google-cloud-sdk-unofficial/google-cloud-sdk/blob/master/install.sh#L133

Reply:
Sounds good!

Reply:
I tried that approach but it didn't work: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/45338/rehearse-45338-pull-ci-cri-o-cri-o-main-ci-cgroupv2-integration/1722391694151782400

Reply:
I think I can use this to fix that issue:
```sh
RUN alternatives --install /usr/bin/python3 python3 /usr/bin/python3.11 1
RUN alternatives --set python3 /usr/bin/python3.11
```

Reply:
can I take this as a follow-up?

Reply:
Sure.